.config/cargo_spellcheck.dic

```diff
@@ -0,0 +1,45 @@
+40
+
+WebAssembly
+Wasm
+validator
+ERC
+fuzzer
+SHA2
+BLAKE2
+KECCAK
+ABI
+FFI
+UI
+crypto
+deploy
+layed
+dispatchable
+SRML
+encodable
+decodable
+spreaded
+callee
+pre
+defrag
+defragments
+growable
+codegen
+endian
+deallocation
+deallocate
+bitvector
+dereferencing
+recurse
+layouting
+accessors
+kb
+
+hasher/S
+hashmap/S
+vector/S
+vec/S
+enum/S
+struct/S
+
+|
```

Comment:
This is in here because we get a number of these for all the doc comment tables otherwise:
```
error: spellcheck(Hunspell)
    --> /home/michi/projects/ink/crates/storage/src/collections/stash/tests.rs:451
     |
 451 |       val |   |   |   |   | E |   |   | H |
     |                                           ^
     |   Possible spelling mistake found.
```

Reply:
oufh ... even with `--checkers hunspell`?

Reply:
if this is the case then the tool kinda seems not markdown-approved.

Reply:
I was able to solve it.